{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>        raise Exception('oops')\nE        Exception: oops\n\ntest_file13.py:13: Exception"}, "start": 1614247805557, "stop": 1614247805557, "uuid": "68af4493-322b-4ea9-b830-192503874b97", "historyId": "aad0c351c33f143b3b2ccda2ec02b206", "testCaseId": "85bf700ec7cf93dcdaecd2c91903cc7a", "fullName": "test_file13#test_broken", "labels": [{"name": "suite", "value": "test_file13"}, {"name": "host", "value": "ZhouKun-Laptop"}, {"name": "thread", "value": "17572-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file13"}]}